desc: --pages-as-heap=yes --massif-out-file=massif.out
cmd: python memory_profile.py
time_unit: i
#-----------
snapshot=0
#-----------
time=0
mem_heap_B=3055616
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=1
#-----------
time=12250754
mem_heap_B=23732224
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=2
#-----------
time=19511302
mem_heap_B=24260608
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=3
#-----------
time=31267645
mem_heap_B=28831744
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=4
#-----------
time=39953737
mem_heap_B=29339648
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=5
#-----------
time=57063304
mem_heap_B=39710720
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=6
#-----------
time=68201577
mem_heap_B=40083456
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=7
#-----------
time=90917190
mem_heap_B=40873984
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=8
#-----------
time=109012003
mem_heap_B=47505408
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=9
#-----------
time=127883533
mem_heap_B=54792192
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=10
#-----------
time=140643441
mem_heap_B=55992320
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=11
#-----------
time=151370375
mem_heap_B=56532992
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=12
#-----------
time=165890950
mem_heap_B=58126336
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 58126336 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n3: 10088448 0x4F48679: mmap (mmap.c:34)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n1: 2895872 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 2895872 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 2105344 0x4ECB182: malloc (malloc.c:2913)
     n1: 2105344 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 2105344 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 2105344 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 2105344 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 2105344 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 2105344 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 2105344 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 2105344 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 2105344 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 2105344 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 2105344 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 2105344 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 2105344 0x4E6782E: (below main) (libc-start.c:291)
    n1: 790528 0x4ECBDC8: calloc (malloc.c:3236)
     n1: 790528 0x4A3C0A: ??? (in /usr/bin/python2.7)
      n1: 790528 0x49F830: PyString_InternInPlace (in /usr/bin/python2.7)
       n2: 790528 0x4BDD67: ??? (in /usr/bin/python2.7)
        n1: 790528 0x4BDE84: ??? (in /usr/bin/python2.7)
         n1: 790528 0x4BE00C: ??? (in /usr/bin/python2.7)
          n1: 790528 0x4BDF0E: ??? (in /usr/bin/python2.7)
           n1: 790528 0x4BDFF6: ??? (in /usr/bin/python2.7)
            n1: 790528 0x4BDC14: PyMarshal_ReadObjectFromString (in /usr/bin/python2.7)
             n1: 790528 0x4BDB94: PyMarshal_ReadLastObjectFromFile (in /usr/bin/python2.7)
              n1: 790528 0x4BDACC: ??? (in /usr/bin/python2.7)
               n1: 790528 0x4BD6A2: ??? (in /usr/bin/python2.7)
                n1: 790528 0x4AFBAB: ??? (in /usr/bin/python2.7)
                 n1: 790528 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                  n1: 790528 0x4B0F76: ??? (in /usr/bin/python2.7)
                   n1: 790528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                    n1: 790528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                     n1: 790528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                      n1: 790528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                       n1: 790528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                        n1: 790528 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                         n1: 790528 0x4BD6EC: ??? (in /usr/bin/python2.7)
                          n1: 790528 0x4AFBAB: ??? (in /usr/bin/python2.7)
                           n1: 790528 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                            n1: 790528 0x4B0F76: ??? (in /usr/bin/python2.7)
                             n1: 790528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                              n0: 790528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
        n0: 0 in 1 place, below massif's threshold (1.00%)
  n3: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n2: 1310720 0x49ED0B: _PyObject_GC_Malloc (in /usr/bin/python2.7)
    n1: 786432 0x4A3A64: _PyObject_GC_NewVar (in /usr/bin/python2.7)
     n0: 786432 in 2 places, all below massif's threshold (1.00%)
    n0: 524288 in 1 place, below massif's threshold (1.00%)
   n1: 786432 0x4AA835: PyString_FromStringAndSize (in /usr/bin/python2.7)
    n1: 786432 0x4BDD0F: ??? (in /usr/bin/python2.7)
     n0: 786432 in 2 places, all below massif's threshold (1.00%)
   n0: 524288 in 1 place, below massif's threshold (1.00%)
 n1: 4317184 0x4F43DD8: brk (brk.c:31)
  n1: 4317184 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 4317184 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 4317184 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 4317184 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 3620864 0x4ECB182: malloc (malloc.c:2913)
       n2: 2572288 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 2297856 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 2297856 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 2297856 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 2297856 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 2297856 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 2297856 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 2297856 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 2297856 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 2297856 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 2297856 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 2297856 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=13
#-----------
time=178468129
mem_heap_B=58798080
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=14
#-----------
time=191938690
mem_heap_B=59473920
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=15
#-----------
time=203094655
mem_heap_B=60014592
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=16
#-----------
time=214178745
mem_heap_B=60555264
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=17
#-----------
time=224507529
mem_heap_B=63057920
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=18
#-----------
time=236040008
mem_heap_B=63598592
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=19
#-----------
time=254030720
mem_heap_B=65323008
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=20
#-----------
time=278284695
mem_heap_B=66371584
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=21
#-----------
time=302292489
mem_heap_B=67420160
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=22
#-----------
time=325057189
mem_heap_B=68468736
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=23
#-----------
time=338032369
mem_heap_B=76861440
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=24
#-----------
time=347686373
mem_heap_B=73711616
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=25
#-----------
time=371680148
mem_heap_B=74760192
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=26
#-----------
time=394326900
mem_heap_B=75808768
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=27
#-----------
time=418176385
mem_heap_B=76857344
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=28
#-----------
time=442797723
mem_heap_B=77905920
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=29
#-----------
time=468053522
mem_heap_B=78954496
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=30
#-----------
time=493926108
mem_heap_B=80003072
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=31
#-----------
time=519318849
mem_heap_B=81051648
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=32
#-----------
time=543253604
mem_heap_B=82100224
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=33
#-----------
time=567722726
mem_heap_B=83148800
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 83148800 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n4: 31055872 0x4F48679: mmap (mmap.c:34)
  n1: 14680064 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 14680064 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 14680064 0x4ECB182: malloc (malloc.c:2913)
     n1: 14680064 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 14680064 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 14680064 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 14680064 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 14680064 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 14680064 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 14680064 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 14680064 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 14680064 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 14680064 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 14680064 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 14680064 0x4E6782E: (below main) (libc-start.c:291)
  n1: 9183232 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 9183232 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 8392704 0x4ECB182: malloc (malloc.c:2913)
     n1: 8392704 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 8392704 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 8392704 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 8392704 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 8392704 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 8392704 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 8392704 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 8392704 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 8392704 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 8392704 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 8392704 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 8392704 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 8392704 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n2: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n1: 1310720 0x49ED0B: _PyObject_GC_Malloc (in /usr/bin/python2.7)
    n0: 1310720 in 2 places, all below massif's threshold (1.00%)
   n0: 1310720 in 2 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=34
#-----------
time=583839053
mem_heap_B=91541504
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 91541504 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n4: 39448576 0x4F48679: mmap (mmap.c:34)
  n1: 17575936 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 17575936 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 16785408 0x4ECB182: malloc (malloc.c:2913)
     n1: 16785408 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 16785408 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 16785408 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 16785408 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 16785408 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 16785408 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 16785408 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 16785408 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 16785408 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 16785408 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 16785408 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 16785408 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 16785408 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 14680064 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 14680064 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 14680064 0x4ECB182: malloc (malloc.c:2913)
     n1: 14680064 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 14680064 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 14680064 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 14680064 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 14680064 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 14680064 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 14680064 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 14680064 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 14680064 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 14680064 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 14680064 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 14680064 0x4E6782E: (below main) (libc-start.c:291)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n2: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n1: 1310720 0x49ED0B: _PyObject_GC_Malloc (in /usr/bin/python2.7)
    n0: 1310720 in 2 places, all below massif's threshold (1.00%)
   n0: 1310720 in 2 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=35
#-----------
time=592587909
mem_heap_B=92585984
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=36
#-----------
time=615511932
mem_heap_B=93634560
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=37
#-----------
time=641268192
mem_heap_B=94683136
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=38
#-----------
time=667443564
mem_heap_B=95731712
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=39
#-----------
time=692337164
mem_heap_B=96780288
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=40
#-----------
time=716428076
mem_heap_B=97828864
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=41
#-----------
time=741224464
mem_heap_B=98877440
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=42
#-----------
time=766608923
mem_heap_B=99926016
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=43
#-----------
time=792512028
mem_heap_B=100974592
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=44
#-----------
time=818798656
mem_heap_B=102023168
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 102023168 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n4: 49930240 0x4F48679: mmap (mmap.c:34)
  n1: 25165824 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 25165824 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 25165824 0x4ECB182: malloc (malloc.c:2913)
     n1: 25165824 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 25165824 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 25165824 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 25165824 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 25165824 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 25165824 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 25165824 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 25165824 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 25165824 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 25165824 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 25165824 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 25165824 0x4E6782E: (below main) (libc-start.c:291)
  n1: 17571840 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 17571840 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 16781312 0x4ECB182: malloc (malloc.c:2913)
     n1: 16781312 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 16781312 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 16781312 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 16781312 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 16781312 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 16781312 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 16781312 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 16781312 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 16781312 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 16781312 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 16781312 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 16781312 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 16781312 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n2: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n1: 1310720 0x49ED0B: _PyObject_GC_Malloc (in /usr/bin/python2.7)
    n0: 1310720 in 2 places, all below massif's threshold (1.00%)
   n0: 1310720 in 2 places, all below massif's threshold (1.00%)
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=45
#-----------
time=845676209
mem_heap_B=103071744
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=46
#-----------
time=873208626
mem_heap_B=104120320
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=47
#-----------
time=899402101
mem_heap_B=105168896
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=48
#-----------
time=927108345
mem_heap_B=106217472
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=49
#-----------
time=952602309
mem_heap_B=107266048
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=50
#-----------
time=977275154
mem_heap_B=108314624
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=51
#-----------
time=1002597357
mem_heap_B=109363200
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=52
#-----------
time=1030165139
mem_heap_B=110411776
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=53
#-----------
time=1056425145
mem_heap_B=111460352
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=54
#-----------
time=1080879571
mem_heap_B=112508928
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 112508928 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n4: 60416000 0x4F48679: mmap (mmap.c:34)
  n1: 35651584 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 35651584 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 35651584 0x4ECB182: malloc (malloc.c:2913)
     n1: 35651584 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 35651584 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 35651584 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 35651584 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 35651584 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 35651584 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 35651584 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 35651584 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 35651584 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 35651584 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 35651584 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 35651584 0x4E6782E: (below main) (libc-start.c:291)
  n1: 17571840 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 17571840 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 16781312 0x4ECB182: malloc (malloc.c:2913)
     n1: 16781312 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 16781312 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 16781312 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 16781312 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 16781312 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 16781312 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 16781312 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 16781312 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 16781312 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 16781312 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 16781312 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 16781312 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 16781312 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n2: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n1: 1310720 0x49ED0B: _PyObject_GC_Malloc (in /usr/bin/python2.7)
    n0: 1310720 in 2 places, all below massif's threshold (1.00%)
   n0: 1310720 in 2 places, all below massif's threshold (1.00%)
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=55
#-----------
time=1096531024
mem_heap_B=146067456
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=56
#-----------
time=1098366215
mem_heap_B=129290240
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=peak
n5: 129290240 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n4: 77197312 0x4F48679: mmap (mmap.c:34)
  n1: 35651584 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 35651584 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 35651584 0x4ECB182: malloc (malloc.c:2913)
     n1: 35651584 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 35651584 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 35651584 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 35651584 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 35651584 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 35651584 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 35651584 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 35651584 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 35651584 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 35651584 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 35651584 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 35651584 0x4E6782E: (below main) (libc-start.c:291)
  n1: 34353152 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 34353152 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 33562624 0x4ECB182: malloc (malloc.c:2913)
     n1: 33562624 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 33562624 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 33562624 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 33562624 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 33562624 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 33562624 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 33562624 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 33562624 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 33562624 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 33562624 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 33562624 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 33562624 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 33562624 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n2: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n1: 1310720 0x49ED0B: _PyObject_GC_Malloc (in /usr/bin/python2.7)
    n0: 1310720 in 2 places, all below massif's threshold (1.00%)
   n0: 1310720 in 2 places, all below massif's threshold (1.00%)
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=57
#-----------
time=1108910285
mem_heap_B=130334720
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=58
#-----------
time=1135751383
mem_heap_B=131383296
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=59
#-----------
time=1162225262
mem_heap_B=132431872
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=60
#-----------
time=1190176750
mem_heap_B=133480448
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=61
#-----------
time=1218935311
mem_heap_B=134529024
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=62
#-----------
time=1245625233
mem_heap_B=135577600
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=63
#-----------
time=1270365598
mem_heap_B=136626176
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=64
#-----------
time=1295061565
mem_heap_B=137674752
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=65
#-----------
time=1320396603
mem_heap_B=138723328
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=66
#-----------
time=1346393513
mem_heap_B=139771904
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 139771904 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n4: 87678976 0x4F48679: mmap (mmap.c:34)
  n1: 46137344 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 46137344 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 46137344 0x4ECB182: malloc (malloc.c:2913)
     n1: 46137344 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 46137344 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 46137344 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 46137344 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 46137344 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 46137344 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 46137344 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 46137344 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 46137344 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 46137344 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 46137344 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 46137344 0x4E6782E: (below main) (libc-start.c:291)
  n1: 34349056 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 34349056 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 33558528 0x4ECB182: malloc (malloc.c:2913)
     n1: 33558528 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 33558528 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 33558528 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 33558528 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 33558528 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 33558528 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 33558528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 33558528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 33558528 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 33558528 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 33558528 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 33558528 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 33558528 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n1: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n0: 2621440 in 3 places, all below massif's threshold (1.00%)
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=67
#-----------
time=1372742049
mem_heap_B=140820480
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=68
#-----------
time=1399547813
mem_heap_B=141869056
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=69
#-----------
time=1426873602
mem_heap_B=142917632
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=70
#-----------
time=1454635853
mem_heap_B=143966208
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=71
#-----------
time=1483015531
mem_heap_B=145014784
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=72
#-----------
time=1510747149
mem_heap_B=146063360
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=73
#-----------
time=1537034723
mem_heap_B=147111936
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=74
#-----------
time=1566408542
mem_heap_B=148160512
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=75
#-----------
time=1594228114
mem_heap_B=149209088
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=76
#-----------
time=1620038356
mem_heap_B=150257664
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 150257664 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n4: 98164736 0x4F48679: mmap (mmap.c:34)
  n1: 56623104 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 56623104 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 56623104 0x4ECB182: malloc (malloc.c:2913)
     n1: 56623104 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 56623104 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 56623104 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 56623104 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 56623104 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 56623104 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 56623104 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 56623104 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 56623104 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 56623104 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 56623104 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 56623104 0x4E6782E: (below main) (libc-start.c:291)
  n1: 34349056 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 34349056 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 33558528 0x4ECB182: malloc (malloc.c:2913)
     n1: 33558528 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 33558528 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 33558528 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 33558528 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 33558528 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 33558528 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 33558528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 33558528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 33558528 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 33558528 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 33558528 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 33558528 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 33558528 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n1: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n0: 2621440 in 3 places, all below massif's threshold (1.00%)
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=77
#-----------
time=1646352071
mem_heap_B=151306240
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=78
#-----------
time=1672780680
mem_heap_B=152354816
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=79
#-----------
time=1699717714
mem_heap_B=153403392
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=80
#-----------
time=1728230951
mem_heap_B=154451968
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=81
#-----------
time=1756190077
mem_heap_B=155500544
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=82
#-----------
time=1782601645
mem_heap_B=156549120
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=83
#-----------
time=1809221208
mem_heap_B=157597696
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=84
#-----------
time=1835638557
mem_heap_B=158646272
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=85
#-----------
time=1862738570
mem_heap_B=159694848
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=86
#-----------
time=1889591784
mem_heap_B=160743424
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=detailed
n5: 160743424 (page allocation syscalls) mmap/mremap/brk, --alloc-fns, etc.
 n4: 108650496 0x4F48679: mmap (mmap.c:34)
  n1: 67108864 0x4EC87AD: sysmalloc (malloc.c:2517)
   n1: 67108864 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 67108864 0x4ECB182: malloc (malloc.c:2913)
     n1: 67108864 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 67108864 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 67108864 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 67108864 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 67108864 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
          n1: 67108864 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
           n1: 67108864 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
            n1: 67108864 0x4F1DED: ??? (in /usr/bin/python2.7)
             n1: 67108864 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
              n1: 67108864 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
               n1: 67108864 0x49E148: Py_Main (in /usr/bin/python2.7)
                n0: 67108864 0x4E6782E: (below main) (libc-start.c:291)
  n1: 34349056 0x4EC8AFD: sysmalloc (malloc.c:2323)
   n2: 34349056 0x4EC9741: _int_malloc (malloc.c:3827)
    n1: 33558528 0x4ECB182: malloc (malloc.c:2913)
     n1: 33558528 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
      n1: 33558528 0x6E3204C: ??? (in /home/ab/pycombina/pycombina.so)
       n1: 33558528 0x6E2F89E: ??? (in /home/ab/pycombina/pycombina.so)
        n1: 33558528 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 33558528 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 33558528 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
           n1: 33558528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
            n1: 33558528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
             n1: 33558528 0x4F1DED: ??? (in /usr/bin/python2.7)
              n1: 33558528 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
               n1: 33558528 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                n1: 33558528 0x49E148: Py_Main (in /usr/bin/python2.7)
                 n0: 33558528 0x4E6782E: (below main) (libc-start.c:291)
    n0: 790528 in 1 place, below massif's threshold (1.00%)
  n1: 4571136 0x4E73825: _nl_load_locale_from_archive (loadarchive.c:228)
   n1: 4571136 0x4E72AF8: _nl_find_locale (findlocale.c:154)
    n1: 4571136 0x4E71FB2: setlocale (setlocale.c:417)
     n1: 4571136 0x49EAAD: Py_InitializeEx (in /usr/bin/python2.7)
      n1: 4571136 0x49DDFE: Py_Main (in /usr/bin/python2.7)
       n0: 4571136 0x4E6782E: (below main) (libc-start.c:291)
  n1: 2621440 0x49EFD2: PyObject_Malloc (in /usr/bin/python2.7)
   n0: 2621440 in 3 places, all below massif's threshold (1.00%)
 n2: 39833600 0x401B479: mmap (mmap.c:34)
  n1: 39366656 0x40068CB: _dl_map_object_from_fd (dl-map-segments.h:56)
   n3: 39366656 0x4008C25: _dl_map_object (dl-load.c:2476)
    n1: 28663808 0x400DBA0: openaux (dl-deps.c:63)
     n1: 28663808 0x4010562: _dl_catch_error (dl-error.c:187)
      n2: 28663808 0x400E1E0: _dl_map_object_deps (dl-deps.c:254)
       n1: 15798272 0x4003A27: dl_main (rtld.c:1647)
        n1: 15798272 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 15798272 0x4001C28: _dl_start (rtld.c:323)
          n1: 15798272 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 15798272 0x0: ???
            n1: 15798272 0xFFEFFFA8D: ???
             n0: 15798272 0xFFEFFFA94: ???
       n1: 12865536 0x40155E0: dl_open_worker (dl-open.c:280)
        n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 12865536 0x4014DA7: _dl_open (dl-open.c:660)
          n1: 12865536 0x5211F07: dlopen_doit (dlopen.c:66)
           n1: 12865536 0x4010562: _dl_catch_error (dl-error.c:187)
            n1: 12865536 0x521256F: _dlerror_run (dlerror.c:163)
             n1: 12865536 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
              n1: 12865536 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
               n1: 12865536 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
                n1: 12865536 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                 n1: 12865536 0x4B0F76: ??? (in /usr/bin/python2.7)
                  n1: 12865536 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                   n1: 12865536 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                    n1: 12865536 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                     n1: 12865536 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                      n2: 12865536 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                       n1: 7000064 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                        n1: 7000064 0x4BD6EC: ??? (in /usr/bin/python2.7)
                         n1: 7000064 0x4AFBAB: ??? (in /usr/bin/python2.7)
                          n1: 7000064 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                           n1: 7000064 0x4B0F76: ??? (in /usr/bin/python2.7)
                            n1: 7000064 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                             n1: 7000064 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                              n0: 7000064 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                       n1: 5865472 0x4F1DED: ??? (in /usr/bin/python2.7)
                        n1: 5865472 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                         n1: 5865472 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                          n1: 5865472 0x49E148: Py_Main (in /usr/bin/python2.7)
                           n0: 5865472 0x4E6782E: (below main) (libc-start.c:291)
    n1: 8597504 0x4015575: dl_open_worker (dl-open.c:237)
     n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 8597504 0x4014DA7: _dl_open (dl-open.c:660)
       n1: 8597504 0x5211F07: dlopen_doit (dlopen.c:66)
        n1: 8597504 0x4010562: _dl_catch_error (dl-error.c:187)
         n1: 8597504 0x521256F: _dlerror_run (dlerror.c:163)
          n1: 8597504 0x5211F9F: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
           n1: 8597504 0x5231C1: _PyImport_GetDynLoadFunc (in /usr/bin/python2.7)
            n2: 8597504 0x522D6D: _PyImport_LoadDynamicModule (in /usr/bin/python2.7)
             n1: 4374528 0x4AF213: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
              n1: 4374528 0x4B0F76: ??? (in /usr/bin/python2.7)
               n1: 4374528 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                n1: 4374528 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                 n1: 4374528 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                  n1: 4374528 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                   n2: 4374528 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                    n1: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                     n1: 2199552 0x4BD6EC: ??? (in /usr/bin/python2.7)
                      n1: 2199552 0x4AFBAB: ??? (in /usr/bin/python2.7)
                       n1: 2199552 0x4AF440: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                        n1: 2199552 0x4B0F76: ??? (in /usr/bin/python2.7)
                         n1: 2199552 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                          n1: 2199552 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                           n1: 2199552 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                            n1: 2199552 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                             n1: 2199552 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                              n0: 2199552 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                    n1: 2174976 0x4F1DED: ??? (in /usr/bin/python2.7)
                     n1: 2174976 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                      n1: 2174976 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                       n1: 2174976 0x49E148: Py_Main (in /usr/bin/python2.7)
                        n0: 2174976 0x4E6782E: (below main) (libc-start.c:291)
             n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
              n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
               n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                 n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                  n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                   n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                    n1: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
                     n1: 4222976 0x4C0619: PyImport_ExecCodeModuleEx (in /usr/bin/python2.7)
                      n1: 4222976 0x4BD6EC: ??? (in /usr/bin/python2.7)
                       n1: 4222976 0x4AFBAB: ??? (in /usr/bin/python2.7)
                        n1: 4222976 0x4AF7E7: PyImport_ImportModuleLevel (in /usr/bin/python2.7)
                         n1: 4222976 0x4B0F76: ??? (in /usr/bin/python2.7)
                          n1: 4222976 0x4B0CB1: PyObject_Call (in /usr/bin/python2.7)
                           n1: 4222976 0x4CE5CE: PyEval_CallObjectWithKeywords (in /usr/bin/python2.7)
                            n1: 4222976 0x4C6ED4: PyEval_EvalFrameEx (in /usr/bin/python2.7)
                             n1: 4222976 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
                              n0: 4222976 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
    n1: 2105344 0x4000EB3: map_doit (rtld.c:499)
     n1: 2105344 0x4010562: _dl_catch_error (dl-error.c:187)
      n1: 2105344 0x40020D4: handle_ld_preload (rtld.c:682)
       n1: 2105344 0x40039AD: dl_main (rtld.c:1539)
        n1: 2105344 0x4019630: _dl_sysdep_start (dl-sysdep.c:249)
         n1: 2105344 0x4001C28: _dl_start (rtld.c:323)
          n1: 2105344 0x4000C36: ??? (in /lib/x86_64-linux-gnu/ld-2.23.so)
           n1: 2105344 0x0: ???
            n1: 2105344 0xFFEFFFA8D: ???
             n0: 2105344 0xFFEFFFA94: ???
  n0: 466944 in 4 places, all below massif's threshold (1.00%)
 n1: 8372224 0x4F43DD8: brk (brk.c:31)
  n1: 8372224 0x4F43EB7: sbrk (sbrk.c:58)
   n1: 8372224 0x4ECE8C7: __default_morecore (morecore.c:47)
    n1: 8372224 0x4EC8633: sysmalloc (malloc.c:2484)
     n2: 8372224 0x4EC9741: _int_malloc (malloc.c:3827)
      n2: 7675904 0x4ECB182: malloc (malloc.c:2913)
       n2: 6627328 0x70C5E76: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.21)
        n1: 6352896 0x6E2F833: ??? (in /home/ab/pycombina/pycombina.so)
         n1: 6352896 0x6E31C79: ??? (in /home/ab/pycombina/pycombina.so)
          n1: 6352896 0x6E2AB45: ??? (in /home/ab/pycombina/pycombina.so)
           n1: 6352896 0x4C4688: PyEval_EvalFrameEx (in /usr/bin/python2.7)
            n1: 6352896 0x4C2763: PyEval_EvalCodeEx (in /usr/bin/python2.7)
             n1: 6352896 0x4C2507: PyEval_EvalCode (in /usr/bin/python2.7)
              n1: 6352896 0x4F1DED: ??? (in /usr/bin/python2.7)
               n1: 6352896 0x4EC650: PyRun_FileExFlags (in /usr/bin/python2.7)
                n1: 6352896 0x4EAE2F: PyRun_SimpleFileExFlags (in /usr/bin/python2.7)
                 n1: 6352896 0x49E148: Py_Main (in /usr/bin/python2.7)
                  n0: 6352896 0x4E6782E: (below main) (libc-start.c:291)
        n0: 274432 in 1 place, below massif's threshold (1.00%)
       n0: 1048576 in 4 places, all below massif's threshold (1.00%)
      n0: 696320 in 3 places, all below massif's threshold (1.00%)
 n0: 3887104 0xFFFFFFFFFFFFFFFF: ???
 n0: 0 in 1 place, below massif's threshold (1.00%)
#-----------
snapshot=87
#-----------
time=1906164186
mem_heap_B=127184896
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
#-----------
snapshot=88
#-----------
time=2110210371
mem_heap_B=122613760
mem_heap_extra_B=0
mem_stacks_B=0
heap_tree=empty
